.idea/artifacts/Homeworks_jar.xml

```diff
@@ -10,6 +10,8 @@
       <element id="extracted-dir" path="$MAVEN_REPOSITORY$/net/bytebuddy/byte-buddy-agent/1.9.3/byte-buddy-agent-1.9.3.jar" path-in-jar="/" />
       <element id="extracted-dir" path="$MAVEN_REPOSITORY$/junit/junit/4.12/junit-4.12.jar" path-in-jar="/" />
       <element id="extracted-dir" path="$MAVEN_REPOSITORY$/net/bytebuddy/byte-buddy/1.9.3/byte-buddy-1.9.3.jar" path-in-jar="/" />
+      <element id="library" level="project" name="Maven: org.hamcrest:hamcrest-core:1.3" />
+      <element id="library" level="project" name="Maven: junit:junit:4.12" />
     </root>
   </artifact>
 </component>
```

Comment:
Im not sure if this file is needed any changes... I think we need to pull only pom.xml with delta changes and FruitShopUnitTest file.

Reply:
Ok.  I just wanted to show changed name of jar file)